submarine-server/server-core/src/main/java/org/apache/submarine/server/notebook/NotebookManager.java

```diff
@@ -79,6 +79,8 @@ public static NotebookManager getInstance() {
    */
   public Notebook createNotebook(NotebookSpec spec) throws SubmarineRuntimeException {
     checkNotebookSpec(spec);
+    String lowerName = spec.getMeta().getName().toLowerCase();
```

Comment:
Does the `experiment name` need to be unified into lowercase?
If necessary, it may be better to control case in the web front end.